src/formatter.ts

```diff
@@ -0,0 +1,156 @@
+import { blueBright, dim, red, reset, underline, yellow } from 'chalk';
+import { ESLint } from 'eslint';
+import stripAnsi from 'strip-ansi';
+import table from 'text-table';
+import { TodoResultMessage } from 'types/estlint-type-extension';
```

Comment:
I'd probably make this a relative import, so it's clear it's local. I'd also explicitly import a `type`.

```suggestion
import type { TodoResultMessage } from '../estlint-type-extension';
```

Reply:
I realized it also had a typo. I changed it to be under `./src/types/index.ts`, I also fixed how it's being imported.